lib/scripts/create_logs_config.py

```diff
@@ -7,14 +7,19 @@
 import os
 import json
 
-LOGS_CONFIG_DIR = os.environ['LOGS_CONFIG_DIR']
-LOGS_CONFIG = os.environ['LOGS_CONFIG']
-DD_TAGS = os.environ['DD_TAGS']
+LOGS_CONFIG_DIR = os.environ.get('LOGS_CONFIG_DIR')
+LOGS_CONFIG = os.environ.get('LOGS_CONFIG')
+DD_TAGS = os.environ.get('DD_TAGS')
```

Comment:
```suggestion
DD_TAGS = os.environ.get('DD_TAGS', [])
```

Reply:
At this stage, we treat `DD_TAGS` as a string, could be comma-separated tags.
We only need to check it's`None` or not, otherwise we just add it to the dict.